views/index.erb

```diff
@@ -0,0 +1,12 @@
+<div style="text-align:center">
+<h1>Rock, paper, scissors!</h1>
+<h2>Sign in to play</h2>
+<form action="/names" method="post">
+  <label for="player_name">
+    Player Name
+    <input type="text" name="player_name">
+  </label>
+  <input type="submit" value="Submit">
+</form>
+<img src="https://pt.pokerdiscover.com/media/blog/2017/01/rock-paper-scissor-ft-1.jpg" class="img-fluid" style="max-width:20%;" alt="...">
+</div>
```

Comment:
I like the layout of the views, I would be keen to see it running as I used buttons to select game outcomes but hadn't thought of using href's